javascript/ql/experimental/adaptivethreatmodeling/src/codeql-suites/javascript-atm-code-scanning.qls

```diff
@@ -1,2 +1,5 @@
 - description: ATM boosted Code Scanning queries for JavaScript
 - queries: .
+- exclude:
+    query filename: StoredXssATM.ql
+    query filename: XssThroughDomATM.ql
```

Comment:
I think it would be safer to explicitly include the four boosted queries here, rather than excluding the ones we don't want to run.